deprecated/class-yoast-modal.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
```

Comment:
The other files in the deprecated directory start with 
```
// Mark this file as deprecated.
_deprecated_file( __FILE__, 'WPSEO 7.0' );
```
Is there a reason to omit this here?